src/include/daos_pool.h

```diff
@@ -70,6 +70,16 @@ struct daos_space {
 	uint64_t		s_total[DAOS_MEDIA_MAX];
 	/** Free space in bytes */
 	uint64_t		s_free[DAOS_MEDIA_MAX];
+	/** Memory file size (in bytes) for md-on-ssd phase2 pool */
+	uint64_t                s_total_mem;
```

Comment:
thanks to Ken for pointing this out. This is actually an API/ABI breaking change since we bring that in daos_space -> daos_pool_space -> daos_pool_info_t; unless i miss something here, the size of pool_info has changed ..
so any app that calls something like pool connect or pool query and passes a non-NULL pool info can potentially break or see memory corruptions if it's not rebuilt with this new library version.
so we do need to bump the major API version if we end up doing this change which would require more testing and re-building all the CI testing apps (a pain). is it possible to delay that change till 3.0?
or is there a way to return that total_mem in a non-breaking ABI way? like a new API to be added for 2.8 then we can make this change for 3.0?

Reply:
yeah, I overlooked that the change to daos_pool_info_t would break API/ABI. I think it's ok to defer this UI improving PR (returning memory file size on pool query for phase2 pool) to 3.0. @tanabarr , what do you think?

Reply:
@NiuYawei Recent changes LGTM. I will address further control-plane comments in a following PR so we can get this landed.